src/wrapper/jnienv.rs

```diff
@@ -1738,7 +1745,11 @@ impl<'local> JNIEnv<'local> {
     }
 
     /// Converts a java byte array to a rust vector of bytes.
-    pub fn convert_byte_array(&self, array: jbyteArray) -> Result<Vec<u8>> {
+    pub fn convert_byte_array<'other_local>(
+        &self,
+        array: &impl AsRef<JByteArray<'other_local>>,
```

Comment:
This method, unlike the others, requires `&impl AsRef<JByteArray>` instead of just `impl AsRef<JByteArray>`. Is that intentional?

Reply:
I don't think so - thanks for pointing out. I can tweak this and re-run CI before landing.